docs/guides/mainnet-forking.md

```diff
@@ -85,6 +89,71 @@ const signer = await ethers.provider.getSigner("0x364d6D0333432C3Ac016Ca832fb859
 signer.sendTransaction(...)
 ```
 
+### Set account nonce
```

Comment:
```suggestion
### Setting account nonce
```
Changing from imperative to present participle, to match other, existing headings.